gcp_airflow_foundations/common/dataflow/jdbc/dataflow_taskgroups.py

```diff
@@ -28,50 +30,49 @@ def dataflow_taskgroup_builder(
 
     create_job_parameters = PythonOperator(
         task_id="create_job_parameters",
-        op_kwargs={
-            "config_params": dataflow_job_params,
-            "table_name": table_name,
-            "destination_table": destination_table,
-            "destination_schema_table": destination_schema_table,
-            "query_schema": query_schema,
-            "owner": dataflow_job_params["database_owner"],
-        },
+        op_kwargs={"config_params": dataflow_job_params,
+                   "table_name": table_name,
+                   "destination_table": destination_table,
+                   "destination_schema_table": destination_schema_table,
+                   "query_schema": query_schema,
+                   "owner": dataflow_job_params["database_owner"]},
         python_callable=create_job_params,
         task_group=taskgroup,
     )
 
     trigger_dataflow_job = PythonOperator(
         task_id="run_dataflow_job_to_bq",
-        op_kwargs={
-            "template_path": dataflow_job_params["template_path"],
-            "system_name": system_name,
-            "table_name": table_name,
-            "query_schema": query_schema,
-        },
+        op_kwargs={"template_path": dataflow_job_params["template_path"],
+                   "system_name": system_name,
+                   "table_name": table_name,
+                   "query_schema": query_schema},
         python_callable=run_dataflow_job,
         task_group=taskgroup,
-        pool=dataflow_job_params["connection_pool"],
+        pool=dataflow_job_params["connection_pool"]
     )
 
     if not query_schema:
-        schema_task_sensor = ExternalTaskSensor(
-            task_id="check_bq_schema_updated",
-            external_dag_id=f"{system_name}_upload_schema",
-            task_group=taskgroup,
-        )
 
         create_table = PythonOperator(
```

Comment:
when you get a chance, look at the new way of doing python operators in Airlflow. Much cleaner